homeassistant/components/sensibo/climate.py

```diff
@@ -373,33 +296,41 @@ async def async_assume_state(self, state):
         else:
             self._external_state = state
 
-    async def async_update(self):
+    async def async_update(self) -> None:
         """Retrieve latest state."""
         try:
             async with async_timeout.timeout(TIMEOUT):
                 data = await self._client.async_get_device(self._id, _FETCH_FIELDS)
         except (
             aiohttp.client_exceptions.ClientError,
             asyncio.TimeoutError,
-            pysensibo.SensiboError,
-        ):
+            SensiboError,
+        ) as err:
             if self._failed_update:
                 _LOGGER.warning(
-                    "Failed to update data for device '%s' from Sensibo servers",
-                    self.name,
+                    "Failed to update data for device '%s' from Sensibo servers with error %s",
+                    self._attr_name,
+                    err,
                 )
-                self._available = False
+                self._attr_available = False
                 self.async_write_ha_state()
                 return
 
-            _LOGGER.debug("First failed update data for device '%s'", self.name)
+            _LOGGER.debug("First failed update data for device '%s'", self._attr_name)
             self._failed_update = True
             return
 
+        self._attr_target_temperature_step = (
```

Comment:
If a ternary spans multiple lines, our standard is to convert it to an `if` block.